README_BUILD.md

```diff
@@ -49,11 +49,13 @@ When importing or creating custom types, the following considerations will allow
 build your types.
 
 * See the notes in Building above.
-* The builder will import your custom types into the vendored resources. They are required to be there for the module to successfully find them during a Puppet catalog application.
+* Do not try to include your module in the import folder by default. When building the first time or during a clean, the builder will delete all files and folders in this directory. It is preferred that you keep those custom modules in a separate location. A suggested location is `build/vendor/custom` within this repository.
```

Comment:
I think we need to sketch out better how we expect users to redistribute their custom content - I think I was coming from the standpoint that they'd reference the module as their fork of the git repo, using r10k.

As an alternative, we could also add a `.gitignore` to prevent any content coming back to us in a PR (I'm not worried about that situation to be honest).